CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (x/gov) [#18173](https://github.com/cosmos/cosmos-sdk/pull/18173) Gov Hooks now returns error and are "blocking" if they fail. Expect for `AfterProposalFailedMinDeposit` and `AfterProposalVotingPeriodEnded` that will log the error and continue.
```

Comment:
Something to be explained when this is done: https://github.com/cosmos/cosmos-sdk/issues/10453